_events/repeated/community-call-planning.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Community Call Planning Meeting
+subtitle:
+location: Virtual
+expires: 2025-12-11
+event_date: "fourth Thursday each month 12:00-13:00 Eastern"
+layout: event
+time:
+  - - start 2022-01-27
+
+# Repeated events information
+repeated: true
+
+# use an rdate string instead (best for complex repeated events)
+# note that the dtstart and rdate at the end are the same
+rrule: 
+  - DTSTART;TZID=America/New_York:20220127T120000
+# fourth Thursday of every month
+  - RRULE:UNTIL=20251225T120000;FREQ=MONTHLY;BYDAY=+4TH
+  - RDATE;TZID=America/New_York:20220127T120000
+---
+
+Planning meetings for USRSE monthly community calls occur on the fourth Thursday of each month. 12:00-13:00 Eastern. This is a working meeting to
```

Comment:
I guess you're skipping Novembers!

Reply:
@mtbc we sort of already have done this for the calls themselves - if there is something else that day we move it/reschedule as needed when we get there etc....